decode/radio_frame.js

```diff
@@ -1,45 +1,43 @@
 var EthernetAddr = require('./ethernet_addr');
 var LogicalLinkControl = require('./llc_packet');
-
+var RadioBeaconFrame = require('./radio_beacon_frame');
 function RadioFrame() {
```

Comment:
the v8 runtime does not care when vars are created. Jshint only whines about it because old js runtimes were fully interpreted and would alloc on each var.

Today like with most modern languages javascript does the same thing regardless of where a var occurs. (And IMO initializing at first use lets tools like jshint provide the more useful warning. Indicating that a variable with that name already exists)